include/deal.II/particles/particle_handler.h

```diff
@@ -1320,13 +1320,14 @@ namespace Particles
   inline void
   ParticleHandler<dim, spacedim>::serialize(Archive &ar, const unsigned int)
   {
-    // Note that we do not serialize the particle data itself. Instead we
-    // use the serialization functionality of the triangulation class, because
-    // this guarantees that data is immediately shipped to new processes if
-    // the domain is distributed differently after resuming from a checkpoint.
-    ar //&particles
-      &global_number_of_particles &global_max_particles_per_cell
-        &next_free_particle_index;
+    Assert(
```

Comment:
Wouldnt this interfere with serializing particle handlers in parallel triangulations? The function seems to suggest that `serialize` is still used for parallel triangulations, but only to serialize all member variables except for the particle data itself. Would we instead need an if-condition that uses the old code for parallel triangulations and your new line 1329 for serial triangulations?